sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -790,11 +790,10 @@ def abort_copy(self, copy_id: Union[str, FileProperties], **kwargs: Any) -> None
 
     @distributed_trace
     def download_file(
-        self, offset=None,  # type: Optional[int]
-        length=None,  # type: Optional[int]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> StorageStreamDownloader
```

Comment:
Did you check async? 😄

Reply:
Crazy how async had it but not sync 🙃 

![Screenshot 2024-11-04 at 12 21 22 PM](https://github.com/user-attachments/assets/6c344210-a97b-4edf-92df-a6250e74ef04)